scenes/game/local_game.gd

```diff
@@ -6159,8 +6526,13 @@ func get_total_power(performing_player : Player, card : GameCard = null):
 	# For now, just assert we're not doing that.
 	assert(performing_player.strike_stat_boosts.power_bonus_multiplier == 1 or performing_player.strike_stat_boosts.power_bonus_multiplier_positive_only == 1)
 
+	var boosted_power = performing_player.strike_stat_boosts.power
+	if performing_player.strike_stat_boosts.power_modify_per_buddy_between:
+		var buddy_count = performing_player.count_buddies_between_opponent()
+		boosted_power += buddy_count * performing_player.strike_stat_boosts.power_modify_per_buddy_between
```

Comment:
hopefully this won't ever have to interact with the positive power multiplier

Reply:
please no.
I'll add a comment that it doesn't work.